packages/web3-providers/src/rss3/index.ts

```diff
@@ -106,6 +113,17 @@ export class RSS3API implements RSS3BaseAPI.Provider, NonFungibleTokenAPI.Provid
             .filter((x) => x.chainId === chainId)
         return createPageable(data, createIndicator(indicator))
     }
+
+    async getWeb3Feed(
+        address: string,
+        { networkPluginId = NetworkPluginID.PLUGIN_EVM }: HubOptions<ChainId> = {},
+        type?: RSS3BaseAPI.FeedType,
+    ) {
+        if (!address) return
+        const url = `${RSS3_FEED_ENDPOINT}account:${address}@${NETWORK_PLUGIN[networkPluginId]}/notes?limit=100&exclude_tags=POAP&tags=Gitcoin&tags=POAP&tags=NFT&tags=Donation&latest=false`
```

Comment:
Please compose it with `urlcat`.

Reply:
this URL has repeated params, `urlcat` can't seem to handle such situation

Reply:
Strange API pararmeting.